inst/@sym/fourier.m

```diff
@@ -17,28 +17,55 @@
 %% If not, see <http://www.gnu.org/licenses/>.
 
 %% -*- texinfo -*-
-%% @deftypefn  {Function File} {@var{FF} =} fourier (@var{f}, @var{x}, @var{k})
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{FF} =} fourier (@var{f}, @var{x}, @var{w})
+%% @deftypefnx {Function File} {@var{FF} =} fourier (@var{f})
+%% @deftypefnx {Function File} {@var{FF} =} fourier (@var{f}, @var{w})
 %% Symbolic Fourier transform.
 %%
-%% FIXME: doc, examples
+%% FIXME: doc
 %%
-%% @seealso{laplace}
+%% Examples:
+%% @example
+%% syms x w
+%% f = exp(-2*abs(x))
+%% fourier(f)
+%% fourier(f, w)
+%% fourier(f, x, w)
+%% @end example
+%%
+%% @seealso{ifourier,laplace}
 %% @end deftypefn
 
 %% Author: Colin B. Macdonald
 %% Keywords: symbolic
 
-function F = fourier(f, x, k)
+function F = fourier(varargin)
+
+  syms f x w;
+  if (nargin == 1)
+    f=varargin{1};
+    x=symvar(f,1);
+
+  elseif (nargin == 2)
+    f=varargin{1};
+    x=symvar(f,1);
+    w=varargin{2};
 
-  if (nargin < 3)
-    syms k
-    warning('todo: check SMT for 2nd argument behavoir');
-  end
+  elseif (nargin == 3)
+    f=varargin{1};
+    x=varargin{2}; 
+    w=varargin{3};
 
-  cmd = { 'd = sp.fourier_transform(*_ins)'
-          'return d,' };
+  else
+    error('Wrong number of input arguments') 
+ 
+  endif
 
-  F = python_cmd (cmd, sym(f), sym(x), sym(k));
+  cmd = { 'sp.FourierTransform._b = -1'
```

Comment:
what is this for?  I cannot find it in docs?  If its important, can you leave a comment here?